sdk/resourcemanager/azure-resourcemanager-resources/src/test/resources/session-records/canCreateResourceGroup.json

```diff
@@ -1,147 +1,157 @@
 {
   "networkCallRecords" : [ {
     "Method" : "PUT",
-    "Uri" : "http://localhost:1234/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/rg39439b?api-version=2020-06-01",
```

Comment:
For resources, I tried to run RECORD and PLAYBACK both. So update record sessions here.